Gemfile.lock

```diff
@@ -1,94 +1,94 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.3.2)
-      actionpack (= 6.0.3.2)
+    actioncable (6.0.3.4)
+      actionpack (= 6.0.3.4)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      activejob (= 6.0.3.2)
-      activerecord (= 6.0.3.2)
-      activestorage (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionmailbox (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      activejob (= 6.0.3.4)
+      activerecord (= 6.0.3.4)
+      activestorage (= 6.0.3.4)
+      activesupport (= 6.0.3.4)
       mail (>= 2.7.1)
-    actionmailer (6.0.3.2)
-      actionpack (= 6.0.3.2)
-      actionview (= 6.0.3.2)
-      activejob (= 6.0.3.2)
+    actionmailer (6.0.3.4)
+      actionpack (= 6.0.3.4)
+      actionview (= 6.0.3.4)
+      activejob (= 6.0.3.4)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.4)
```

Comment:
### [actionpack](https://rubyonrails.org)

**Hotfix** version upgrade :chart_with_upwards_trend::small_red_triangle: 6.0.3.2 → 6.0.3.4

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.4/actionpack)_]

:tada: Patched vulnerabilities:

- [CVE-2020-8264](https://nvd.nist.gov/vuln/detail/CVE-2020-8264)
  **Possible XSS Vulnerability in Action Pack in Development Mode**

  
  URL: https://groups.google.com/g/rubyonrails-security/c/yQzUVfv42jk

  There is a possible XSS vulnerability in Action Pack while the application server is in development mode.  This vulnerability is in the Actionable Exceptions middleware.  This vulnerability has been assigned the CVE identifier CVE-2020-8264.  Versions Affected:  >= 6.0.0 Not affected:       < 6.0.0 Fixed Versions:     6.0.3.4  Impact ------ When an application is running in development mode, and attacker can send or embed (in another page) a specially crafted URL which can allow the attacker to execute JavaScript in the context of the local application.  Workarounds ----------- Until such time as the patch can be applied, application developers should disable the Actionable Exceptions middleware in their development environment via a line such as this one in their config/environment/development.rb:  `config.middleware.delete ActionDispatch::ActionableExceptions`






<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.3.2...v6.0.3.4).

These are the individual commits:
- (4ca2027) [Fix XSS vulnerability in `translate` helper](https://github.com/rails/rails/commit/4ca2027d70ddbab141300e2b8331d2d775c63154)
- (070d4af) [v6.0.3.3](https://github.com/rails/rails/commit/070d4afacd3e9721b7e3a4634e4d026b5fa2c32c)
- (bd5d907) [Prevent XSS in the Actionable Exceptions middleware](https://github.com/rails/rails/commit/bd5d907e72fe799f5455804843a92695a5f29e63)
- (fe76a95) [Preparing for 6.0.3.4 release](https://github.com/rails/rails/commit/fe76a95b0d252a2d7c25e69498b720c96b243ea2)


</details>